mumbleBot.py

```diff
@@ -653,6 +658,23 @@ def resume(self):
         self.wait_for_ready = True
         self.pause_at_id = ""
 
+    # map the volume_before_converting(0~1) into -60~0 dB, convert dB into a factor between 0~1.
+    def convert_set_volume(self, volume_before_converting):
+        self.unconverted_volume = volume_before_converting
+        dB = -60 + volume_before_converting * 60
+        self._volume_set = 10 ** (dB / 20)
+
+    def get_displayed_volume(self):
+        return self.unconverted_volume
+
+    def convert_set_ducking_volume(self, volume_before_converting):
```

Comment:
Is it possible to combine the function that computes the volume transform into one function? Because currently `convert_set_volume` and `convert_set_ducking_volume` are mostly the same code :)

Reply:
I'm separating the volume part into a helper, that will make the code cleaner.